docs/spectrum1d.rst

```diff
@@ -126,8 +126,8 @@ common spectral axis.
 
     >>> spec = Spectrum1D(spectral_axis=np.arange(5000, 5010)*u.AA, flux=np.random.sample((5, 10))*u.Jy)
     >>> spec_slice = spec[0] #doctest:+SKIP
-    >>> spec_slice.wavelength #doctest:+SKIP
-    <Quantity [0., 1., 2., 3., 4., 5., 6., 7., 8., 9.] Angstrom>
+    >>> spec_slice.spectral_axis #doctest:+SKIP
```

Comment:
Lines 25, 26, and 100 also still make reference to `wavelength` which should probably be changed to `spectral_axis`

Reply:
Lines 25/26 can stay `wavelength` since we only want to express that `spectral_axis` should be used while _interacting_ with the `Spectrum1D` object.